sample/build.gradle

```diff
@@ -4,6 +4,8 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'com.google.gms.google-services'
 apply plugin: 'org.jlleitschuh.gradle.ktlint'
+apply plugin: "com.hiya.jacoco-android"
```

Comment:
Why do we need jacoco into our sample app?

Reply:
we dont